src/waveform/renderers/allshader/waveformrendererstem.cpp

```diff
@@ -14,6 +14,17 @@
 #include "waveform/renderers/waveformwidgetrenderer.h"
 #include "waveform/waveform.h"
 
+namespace {
+#ifdef __SCENEGRAPH__
+// FIXME this is a workaround an issue with waveform only drawing partially in
+// SG. The workaround is to reduce the the number of vertices, by reducing the
+// precision of waveform strips.
+const float kPixelPerStrip = 2;
+#else
+const float kPixelPerStrip = 1;
+#endif
+} // namespace
```

Comment:
file an issue after merge so we don't forget?

Reply:
Yep, good call - created as #14407